src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/EntityAIRangeGuard.java

```diff
@@ -101,11 +101,6 @@
     private static final double AIM_HEIGHT = 3.0D;
 
     /**
-     * Experience the guard receives each shot arrow.
-     */
-    private static final double XP_EACH_ARROW = 0.2;
-
-    /**
      * Used to calculate the chance that an arrow hits, if the worker levels is higher than 15 the chance gets worse again.
      * Because of the rising fire speed.
      */
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "XP_EACH_ARROW" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)